components/breadcrumb/demo/router-4.md

```diff
@@ -0,0 +1,75 @@
+---
+order: 3
+iframe: 200
+reactRouter: react-router-dom
```

Comment:
Actually, react-router@4 and react-router-dom@4 are different.

And we should use react-router-dom in browser.